examples/counter/common/build.gradle

```diff
@@ -0,0 +1,97 @@
+repositories {
+    maven { url "https://dl.bintray.com/spekframework/spek-dev" }
+}
+apply plugin: 'java'
+apply plugin: 'kotlin-multiplatform'
+
+
+kotlin {
+    jvm()
+    js() {
+        [compileKotlinJs, compileTestKotlinJs].each { configuration ->
+            configuration.kotlinOptions {
+                moduleKind = 'umd'
+                sourceMap = true
+                metaInfo = true
+            }
+        }
+    }
+
+
+    iosArm64("ios")
+    iosX64("iosSim")
+    macosX64("macos")
+    mingwX64("win")
+    wasm32("wasm")
+    linuxArm32Hfp("linArm32")
+    linuxMips32("linMips32")
+    linuxMipsel32("linMipsel32")
+    linuxX64("lin64")
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation kotlin("stdlib-common")
+                implementation project(":lib")
+            }
+        }
+        commonTest {
+            kotlin.srcDir('src/test/kotlin')
+            dependencies {
+                implementation kotlin("test-common")
+                implementation kotlin("test-annotations-common")
+                implementation "org.spekframework.spek2:spek-dsl-metadata:$project.versions.spek"
+                implementation "ch.tutteli.atrium:atrium-cc-en_GB-robstoll-common:$project.versions.atrium"
+                implementation "io.mockk:mockk-common:1.9.3"
+            }
+        }
+
+        jvmMain {
+            kotlin.srcDir('src/jvmMain/kotlin')
+            dependencies {
+                implementation kotlin("stdlib")
+            }
+        }
+        jvmTest {
+            dependencies {
+                implementation kotlin("test")
+                implementation kotlin("test-junit")
+                implementation "org.spekframework.spek2:spek-dsl-jvm:$project.versions.spek"
```

Comment:
I've had a good amount of success with [buildSrcVersions](https://github.com/jmfayard/buildSrcVersions) for generating code to manage dependencies across modules. Will also give you type completion in the ide, once the buildSrc module is set up

Reply:
I've been looking at that and looks nice.  Want to make a ticket for updating on trello? - either to buildSrcVersions, or at least a standard across reduxkotlin, reduxkotlin-thunk, etc.   The other repos use a slightly different strategy

Reply:
Done